libraries/input-plugins/src/input-plugins/KeyboardMouseDevice.cpp

```diff
@@ -113,9 +113,16 @@ void KeyboardMouseDevice::eraseMouseClicked() {
     _inputDevice->_buttonPressedMap.erase(_inputDevice->makeInput(Qt::RightButton, true).getChannel());
 }
 
-void KeyboardMouseDevice::mouseMoveEvent(QMouseEvent* event) {
+void KeyboardMouseDevice::mouseMoveEvent(QMouseEvent* event, bool capture, QPointF captureTarget) {
     QPoint currentPos = event->pos();
 
+    if (!capture) {
+        _accumulatedMove += currentPos - _lastCursor;
+    } else if (!isNaN(captureTarget.x())) {
+        QPointF change = event->globalPos() - captureTarget;
+        _accumulatedMove += QPoint(change.x(), change.y());
```

Comment:
If `mouseMoveEvent()` calls happen more often than calls to `pluginUpdate()` then the amount of movement will be multiplied?

Reply:
not sure what you mean.  it’s additive, so this _accumulatedMove will get larger if multiple mouse moves happen at once.  perhaps this explains the framerate dependent behavior that Julian is seeing, but it’s equivalent to how it’s done in master now (the difference is just calculated here instead of above)

Reply:
OK